### DISCUSSION TITLE
Can I clear all formatting?

### DISCUSSION THREAD

**Discussion by kubinka0505** [Textualize/rich#3095] [Q&A]
How can I reset the text formatting in `rich.rprint` **not** using defined style?
```py
from rich import print
print("[bold red]a text[/end]")
```
As visible I've **not** used the style in the text — I am looking for something *global*.

I've also tried with `rich.markdown` and `<b><font color=red>a text</font></b>`, but with no results besides text printed after 2 new lines.

**Comment by willmcgugan** [comment]
I'm afraid I don't understand your question. Have you read the docs on Console markup? https://rich.readthedocs.io/en/latest/markup.html

**Reply by kubinka0505** [reply]
> I'm afraid I don't understand your question. Have you read the docs on Console markup? https://rich.readthedocs.io/en/latest/markup.html

Thanks for tip, now I have. 🙂

![Screenshot_20230820_124644.jpg](https://github.com/Textualize/rich/assets/30776937/18c37683-48f5-411f-b635-6ab6c75cbaaf)
